src/use_case/matching/logout/LogoutInteractor.java

```diff
@@ -0,0 +1,19 @@
+package use_case.matching.logout;
+
+public class LogoutInteractor implements LogoutInputBoundary {
+    final LogoutUserDataAccessInterface userDataAccessObject;
+    final LogoutOutputBoundary logoutPresenter;
+
+    public LogoutInteractor(LogoutUserDataAccessInterface userDataAccessInterface,
+                           LogoutOutputBoundary logoutOutputBoundary) {
+        this.userDataAccessObject = userDataAccessInterface;
+        this.logoutPresenter = logoutOutputBoundary;
+    }
+    @Override
+    public void execute() {
+       // String loggedoutUser = userDataAccessObject.get(loginInputData.getUsername());
```

Comment:
Is this code redundant? I'm not sure why it is commented out here

Reply:
Yes, I forgot to delete this line of code as it was no longer necessary for my implementation. Thank you!

Reply:
Ok great!